site/src/contexts/ProxyContext.test.tsx

```diff
@@ -352,22 +352,14 @@ describe("ProxyContextSelection", () => {
 
       // Mock the API response
       server.use(
-        rest.get("/api/v2/regions", async (req, res, ctx) => {
-          return res(
-            ctx.status(200),
-            ctx.json({
-              regions: regions,
-            }),
-          );
-        }),
-        rest.get("/api/v2/workspaceproxies", async (req, res, ctx) => {
-          return res(
-            ctx.status(200),
-            ctx.json({
-              regions: regions,
-            }),
-          );
-        }),
+        http.get("/api/v2/regions", () =>
+          HttpResponse.json({
+            regions,
+          }),
+        ),
+        http.get("/api/v2/workspaceproxies", async () =>
```

Comment:
Does this function still need to be async?

Reply:
I don't think so. Probably old stuff.